src/pretix/control/templates/pretixcontrol/checkin/fragment_checkin_source_type.html

```diff
@@ -0,0 +1,7 @@
+{% load i18n %}
+
+{% if checkin.raw_source_type == "barcode" %}
+    <span class="fa fa-qrcode fa-fw" data-toggle="tooltip_html" title="{% trans "User presented a QR-Code" %}"></span>
+{% elif checkin.raw_source_type == "nfc_uid" %}
```

Comment:
What about nfc_mf0aes?

Reply:
Fair point... I haven't had a single nfc_mf0aes yet in my database as a checkin-object yet... Hmhmhm... Let's see if we can somehow come up with a teeny-tiny icon that represents this and is still legible...

Reply:
Changed this to use BaseMediaType similar to the SalesChannels.

Added a new Icon base on a single letter (U = UID, A = AES) plus `))`...